citi-onboarding/Recorda-one-page
# Feature/secao inicial

What I did:
- Created Initial Section Page
- Created Data component 
-  Created Tela de Celular component
- Deleted the Home page and everything releared to that (The not important ones)

How to test:
- Git checkout main
- Git pull
- Git checkout feature/secao_inicial
- Open the "client" directory at the terminal
- Write "npm run start" at the terminal
- Check if the Initial Section is working
- Review the code
